modules/ctl/src/main/scala/hi/deploy.scala

```diff
@@ -9,8 +9,9 @@ import gem.ctl.low.git._
 import gem.ctl.low.docker._
 import gem.ctl.hi.common._
 
-import scalaz._, Scalaz._
+import cats.implicits._
 import scala.util.matching.Regex
+import mouse.all._
```

Comment:
Some minor syntax enhancements like `"123".parseInt` that are standard in scalaz are not provided by cats, so that's what mouse does. I'm pushing to merge it with cats for 1.0 though. It's kind of dumb.